web/cobrands/sass/_base.scss

```diff
@@ -2499,8 +2499,16 @@ label .muted {
 }
 
 a#geolocate_link {
-  display: inline-block;
-  padding: 0.5em;
+  display: inline-block !important;
+  border-radius: 6px;
+  border: 1px solid #333333;
+  color: #222222;
+  text-decoration: none;
+  font-weight: bold;
+  background: transparent url("/i/locate-me-arrow-black.svg") 4% 50% no-repeat;
```

Comment:
Your `background` and `background-image` lines here are the wrong way round – the svg image will never get used, because it’s overwritten with the png.

Also, we have a mixin to output the two background-image lines. Here’s how you’d use it.

```background: transparent;
background-position: 0 0;
background-repeat: no-repeat;
background-size: 24px 24px;
@include svg-background-image('/i/locate-me-arrow-black');
```

A shame we have to include all the other `-position`, `-repeat`, etc bumph – maybe something we could improve in future!

Another thing I wonder is whether we could inline the SVG, into the HTML of the button itself. That way, if we give the SVG `{ fill: currentColor }` (or `fill="currentColor"`) then it’d automatically inherit the same colour as the button text, which would save us generating a new image for each cobrand with slightly different button text colours?